packages/pluggableWidgets/badge-web/src/components/__tests__/Badge.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import { shallow, ShallowWrapper } from "enzyme";
+import { createElement } from "react";
+
+import { Badge, BadgeProps } from "../Badge";
+
+describe("Badge", () => {
+    const createBadge = (props: BadgeProps): ShallowWrapper<any, any> => shallow(<Badge {...props} />);
+
+    let defaultBadgeProps: BadgeProps;
+
+    beforeEach(() => {
+        defaultBadgeProps = {
+            type: "badge",
+            className: "custom-class",
+            style: { padding: 5 },
```

Comment:
better remove the style from default snapshots and move it to a test case. Same for `className`

Reply:
Done.